test/testdrive/esoteric/s3-gzip.td

```diff
@@ -23,7 +23,7 @@ b2
 b3
 
 > CREATE MATERIALIZED SOURCE s3_all_none
-  FROM S3 OBJECTS FROM SCAN BUCKET '${bucket}'
+  FROM S3 OBJECTS FROM SCAN BUCKET 'testdrive-${bucket}'
```

Comment:
Minor nit: this bucket name will now be "testdrive-materialize-ci-testdrive-none-${testdrive.seed}". Perhaps we should remove the long prefixes from the testdrive bucket names now that testdrive formats the bucket name with the "testdrive" prefix?

Reply:
Oops, yeah, meant to do that! Thanks for the reminder.